src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -14,6 +15,15 @@
 logger = logging.getLogger(__name__)
 
 
+class TableMigrationType(Enum):
```

Comment:
```suggestion
class Scope(Enum):
```
it's more of a scope, not a type.